Sources/ComposableEnvironment/DerivedEnvironment.swift

```diff
@@ -48,7 +48,20 @@ public final class DerivedEnvironment<Value> where Value: ComposableEnvironment
     }
   }
 
-  var environment: Value
+  var _environment: Value?
+  var environment: Value {
+    set { _environment = newValue }
+    get {
+      if let environment = _environment {
+        return environment
+      } else {
```

Comment:
Totally agree, just forgot to add it 🌚

Reply:
Oh, I thought I forgot... 🤡
I prefer to always if-else or guard otherwise. But I can remove else or put a guard, your choice?

Reply:
If we go the `lazy` way ultimately, this code will go away.

Otherwise, the `guard` would be counter intuitive in this case. Replacing `if () { return } else {}` by `if () { return }…` is the prefered variant in other languages like Go for example. 
You can keep the `else`, this is an extremely minor concern.

But this is mostly a matter of taste, and these discussions are by definition without any clear resolution. The best will be to set up some CI action to automatically enforce some standard with `swift-format`.